### PR TITLE
support tags in domain spans

### PR DESCRIPTION
**Pull request by Sovianum**
Tags were supported in storage spans but were not passed to the domain level. Fixing it in current PR

### CODE REVIEW

File: lib/src/main/scala/com/joom/trace/analysis/spark/TraceConverter.scala [hunk 1]
```diff
@@ -153,13 +153,20 @@ object TraceConverter {
   }
 
   private def spanFromStorageSpan(span: Storage.Span, executionGroups: Seq[Domain.ExecutionGroup], spanTimes: SpanTimes): Domain.Span = {
+
+    val tags = span.tags match {
+      case None => Seq()
+      case Some(tags) => tags.map(tag => Domain.Tag(tag.key, tag.vStr))
+    }
```
Comment (vprus) [inline lib/src/main/scala/com/joom/trace/analysis/spark/TraceConverter.scala:160]: Sounds like
```
val tags =  span.tags.getOrElse(Seq()).map(tag => Domain.Tag(tag.key, tag.vStr)))
````
might be more compact. In fact, do we really need Storage.Span.tags to be Option[Seq]? Can it be just Seq?

**Review by vprus — COMMENTED** [review]
I have a minor suggestion, but the PR is OK regardless of what you decide.

File: lib/src/main/scala/com/joom/trace/analysis/spark/TraceConverter.scala (hunk 1 above)
Comment (Sovianum) [reply lib/src/main/scala/com/joom/trace/analysis/spark/TraceConverter.scala:160]: I think, you are right; Storage.Stan.tags better be Seq

Comment (Sovianum) [reply lib/src/main/scala/com/joom/trace/analysis/spark/TraceConverter.scala:160]: I tried to change all Option[Seq] to just Seq and some of my tests failed with null pointer exception. So for now i leave it with options

**Pull request merged**